snap/gui/musescore.desktop

```diff
@@ -0,0 +1,17 @@
+[Desktop Entry]
```

Comment:
Just a quick note for anybody who might decide to pick this up. This file is basically the same as [build/Linux+BSD/mscore.desktop.in](https://github.com/musescore/MuseScore/blob/102fe1c4876ac0e410b3d61883784536bd780efa/build/Linux%2BBSD/mscore.desktop.in) with just a few slight (but important) differences. The two files should probably be combined into a single file, with those differences inserted at build time by CMake.